aws-auth.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+data:
+  mapRoles: |
+    - groups:
+      - system:bootstrappers
+      - system:nodes
+      rolearn: arn:aws:iam::905418087090:role/eksctl-mpm-nodegroup-standard-work-NodeInstanceRole-u9O9QvFivBjm
+      username: system:node:{{EC2PrivateDNSName}}
+  mapUsers: |
+    - userarn: arn:aws:iam::905418087090:user/daniel
+      username: daniel
+      groups:
+        - system:masters
+    - userarn: arn:aws:iam::905418087090:user/yannick
+      username: yannick
+      groups:
+        - system:masters
```

Comment:
Could we find a way to have the re-usable? I.e. without hardcoding AWS account id's and user accounts here?